index.js

```diff
@@ -99,6 +100,11 @@ module.exports = function (text, opts) {
 
 	var NL = '\n';
 	var PAD = ' ';
+
+	if (opts.align && opts.align !== 'left') {
+		text = ansiAlign(text, {align: opts.align});
+	}
```

Comment:
Ok - fortunately I know the author. 😄 My original rationale was "if you don't need center- or right- alignment, then don't call `ansi-align`", but I see now that was probably a bit short-sighted. Let me see what I can do.